core-web/libs/contenttype-fields/src/lib/fields/binary-field/components/dot-ui-message/dot-ui-message.component.scss

```diff
@@ -0,0 +1,31 @@
+@use "variables" as *;
+
+:host {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: $spacing-3;
+    height: 100%;
+    padding: $spacing-3;
+
+    .icon-container {
+        border-radius: 50%;
+        padding: $spacing-3;
+
+        &.info {
+            color: $color-palette-secondary-500;
+            background: $color-palette-secondary-200;
+        }
+
+        &.error {
+            color: $color-alert-yellow;
+            background: $color-alert-yellow-light;
+        }
+    }
+
+    .text {
+        text-align: center;
+        line-height: 140%;
+    }
```

Comment:
We don't need this here because it will result in `[sd3s] .icon-container[as5s]` but onlt this `.icon-container[as5s]` will be enough.

Reply:
Done! Thank you!